### PR TITLE
[MOBILE-198] add pr and issue templates

### PR DESCRIPTION
**Pull request by crow**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+
+# Please fill out this template when submitting an pull request.
```
Comment (BrianBatchelder) [inline .github/PULL_REQUEST_TEMPLATE.md]: ~an~ _a_

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+# Please fill out this template when submitting an issue.
+All lines beginning with an ℹ symbol indicate information that's important for you to provide to ensure your issue is resolved as quickly and efficiently as possible.
+
+❗For how-to inquiries involving Urban Airship functionality or use cases, please
+contact (support)[https://support.urbanairship.com/hc/en-us].
+
+# Preliminary Info
+
+### What Urban Airship dependencies are you using?
+ℹ Please include Urban Airship dependencies and their versions here.
```
Comment (rlepinski) [inline .github/ISSUE_TEMPLATE.md]: What version of urbanairship-cordova are you using?

Comment (rlepinski) [reply .github/ISSUE_TEMPLATE.md]: We also need to know cordova version, cordova-ios version, cordova-android version and what platform are they experiencing the issue on

**Comment by rlepinski** [comment]
👍 after a few updates

**Pull request merged**